docs/blog/_posts/2021-02-17-scala3-rc1.md

```diff
@@ -0,0 +1,242 @@
+- Fix [#10863](https://github.com/lampepfl/dotty/issues/10863): Make show `AnyKind`ed [#10988](https://github.com/lampepfl/dotty/pull/10988)
+- Add ParamClause to allow multiple type param clauses [#11074](https://github.com/lampepfl/dotty/pull/11074)
+- Rework reflect Symbol fields API [#10705](https://github.com/lampepfl/dotty/pull/10705)
+- Rename `Liftable` to `ToExpr` and `Unliftable` to `FromExpr` [#10618](https://github.com/lampepfl/dotty/pull/10618)
+- Expand non-transparent macros after Typer [#9984](https://github.com/lampepfl/dotty/pull/9984)
+- Rework TastyInspector API to allow inspection of all files [#10792](https://github.com/lampepfl/dotty/pull/10792)
+- Allow leading context parameters in extension methods [#10940](https://github.com/lampepfl/dotty/pull/10940)
+- Rename `Not` to `NotGiven` to make its purpose clearer [#10720](https://github.com/lampepfl/dotty/pull/10720)
+- Fix [#10709](https://github.com/lampepfl/dotty/issues/10709): Add missing level check before inlining [#10781](https://github.com/lampepfl/dotty/pull/10781)
+
+# Let us know what you think!
+If you have questions or any sort of feedback, feel free to send us a message on our
+[Gitter channel](https://gitter.im/lampepfl/dotty). If you encounter a bug, please
+[open an issue on GitHub](https://github.com/lampepfl/dotty/issues/new).
+
+
+## Contributors
+Thank you to all the contributors who made this release possible 🎉
+
+According to `git shortlog -sn --no-merges 3.0.0-M3..3.0.0-RC1` these are:
+
+```
+   183  Martin Odersky
+   138  Nicolas Stucki
+    36  Krzysztof Romanowski
+    25  Filip Zybała
+    25  Liu Fengyun
+    24  Lan, Jian
+    22  Jamie Thompson
+    19  Tom Grigg
+    17  Andrzej Ratajczak
+    16  Stéphane Micheloud
+    15  Guillaume Martres
+    11  Paweł Marks
+     9  Phil
+     6  Aleksander Boruch-Gruszecki
+     6  Jonathan Brachthäuser
+     6  Natsu Kagami
+     6  odersky
+     4  Jasper Moeys
+     4  Adrien Piquerez
+     3  Sébastien Doeraene
+     3  Michał Pałka
+     3  Albert Chen
+     2  Alexandre Archambault
+     2  Som Snytt
+     2  kenji yoshida
+     2  Luc Henninger
+     2  Ayush
+     2  Raphael Jolly
+     2  Anatolii Kmetiuk
+     2  Olivier Blanvillain
+     2  changvvb
+     1  ysthakur
+     1  Ang Hao Yang
+     1  Ang9876
+     1  AngAng
+     1  August Nagro
+     1  Ciara O'Brien
+     1  Dale Wijnand
+     1  Florian Cassayre
+     1  Florian Schmaus
+     1  Iltotore
+     1  Jason Zaugg
+     1  Julien Richard-Foy
+     1  Katrix
+     1  Master-Killer
+     1  Michael Pilquist
+     1  Mikael Blomstrand
+     1  Mike Samuel
+     1  Philippus
+     1  Philippus Baalman
+     1  Rick M
+     1  Stephane MICHELOUD
+     1  Timur Abishev
+     1  Tomas
+     1  ansvonwa
+     1  ayush
+     1  costa100
+     1  iroha168
+     1  noti0na1
+     1  riiswa
+     1  tanishiking
+```
+
+If you want to get your hands dirty and contribute to Dotty, now is a good time to get involved!
```

Comment:
Here and below, Dotty is mentioned several times. Search in the entire blog post for Dotty and replace with Scala 3.